ansible/app.yml

```diff
@@ -1,18 +1,18 @@
-- hosts: all
-  become: true
-  roles:
-    - aeriscloud.wkhtmltopdf
-    - geerlingguy.mysql
-    - mysql
```

Comment:
j'avoue :)
sauf la syntaxe yml avec les "=" :)

`- mysql_db:
    state:  import
    name:   yprox
    target: /srv/database/yprox.sql.gz
  when: db_reset|bool == true and db.stat.exists == true
`